client/src/modules/inventory/list/templates/action.cell.html

```diff
@@ -14,6 +14,13 @@
         <i class="fa fa-edit"></i> <span translate>FORM.BUTTONS.EDIT</span>
       </a>
     </li>
+    <li>
+      <a href
+        ng-click="grid.appScope.inventoryLogModal(row.entity.uuid)"
+        data-method="log">
+        <span translate>FORM.LABELS.CHANGES</span>
+      </a>
```

Comment:
Is it possible to add an icon for this link.

Reply:
![image](https://user-images.githubusercontent.com/5445251/58807969-fe909e00-8610-11e9-87c7-81fd4d4e21ec.png)

👆